### PR TITLE
Update Developing section in README

### PR DESCRIPTION
**Pull request by gbvivian**
Developing section now specifies where scheduler-backend file will be

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ Retrieves and uses data from UBC Course Schedule.
...
 Make sure you have `Go`, `make`, `Docker` installed
 
-Get the code (to your `$GOPATH`):
+To get the code (to your `$GOPATH`):
 ```
 go get -d github.com/smart-cs/scheduler-backend
 ```
+Your repository should end up in ~/go/src/github.com/smart-cs/scheduler-backend
 
 Run the server locally:
```
Comment (nickwu241) [inline README.md]: Let's also make this `To run the server locally` to stay consistent?

File: README.md [hunk 2]
```diff
@@ -4,10 +4,11 @@ Retrieves and uses data from UBC Course Schedule.
 ## Developing
 Make sure you have `Go`, `make`, `Docker` installed
 
-Get the code (to your `$GOPATH`):
+To get the code (to your `$GOPATH`):
 ```
 go get -d github.com/smart-cs/scheduler-backend
 ```
+Your repository should end up in ~/go/src/github.com/smart-cs/scheduler-backend
```
Comment (nickwu241) [inline README.md]: The default `$GOPATH` is `~/go`, what do you think about something like:

The default `$GOPATH` is `~/go` so the repository should end up in `~/go/src/github.com/smart-cs/scheduler-backend`, assuming your `$GOPATH` is `~/go`.

Comment (gbvivian) [reply README.md]: How about:
By default, your $GOPATH is ~/go. If you haven't changed it, the repository should end up in ~/go/src/github.com/smart-cs/scheduler-backend

Comment (nickwu241) [reply README.md]: Sound good to me, you can wrap $GOPATH, and the directories with back ticks -- ` to show it's code.

Comment (gbvivian) [reply README.md]: `test`

**Pull request merged**